README.md

```diff
@@ -257,15 +267,45 @@ Building the program with BLAS support may lead to some performance improvements
     cmake --build . --config Release
     ```
 
-### Prepare Data & Run
+### Aquiring Setting up the 7b model weights
+
+You can use this system to conduct research on an AI chatbot vaguely comparable to ChatGPT-3 and it will even run on your local machine without needing massive amounts of hardware.  But to do so you **must** install the Alpaca 7b model weights into the models folder.
+
+Because these resources belong to Facebook, their official path to obtaining the data should be followed.  While it's true that most researchers using the Alpaca weights obtained them from a magnet link to a torrent file, linking or sharing that magnet link should not be done in this repo due to the questionability of violating FaceBook's IP rights and also (not to be an alarmist here) the potential for the popularization of these weights to cause harm.
+
+- **Under no circumstances should IPFS, magnet links, or any other links to model downloads be shared anywhere in this repository, including in issues, discussions, or pull requests. They will be immediately deleted.**
+- The LLaMA models are officially distributed by Facebook and will **never** be provided through this repository.
+- Refer to [Facebook's LLaMA repository](https://github.com/facebookresearch/llama/pull/73/files) if you need to request access to the model data.  Any Magnet links shared
+
+#### Putting the Model Weights in the Right Spot
+
+This guide will assume that you've downloaded the files to an arbitrary folder, `/mnt/c/ai/models/LLaMA` using some responsible means described above.
```

Comment:
This path is oddly specific. If we do need a specific path, it might be best to use something a little more platform agnostic and, at the very least, drop the `/mnt/c` part.

Reply:
Agreed, it's specific to using WSL2.  I had to do a little research to find the path to my Windows C drive, but I'll correct this with `/path/to/LLaMA` (assuming documenting the LLaMA folder's name is of value per above).